File: praaktic_sdk/src/main/java/com/praaktis/exerciseengine/Engine/DetailPoint.java
package com.praaktis.exerciseengine.Engine;

import java.io.Serializable;

public class DetailPoint implements Serializable {
    public float value;
    public int id;

    public DetailPoint(float value, int id) {
        this.value = value;
        this.id = id;
    }
}

File: app/src/main/java/com/mobile/gympraaktis/domain/entities/DetailPoint.kt
package com.mobile.gympraaktis.domain.entities

import com.google.gson.annotations.SerializedName
import java.io.Serializable

data class DetailPoint(
    @SerializedName("id")
    val id: Int,
    @SerializedName("name", alternate = ["label"])
    val name: String,
    @SerializedName("max_value")
    val maxValue: Float,
    @SerializedName("help_text")
    val helpText: String?,
) : Serializable